### ISSUE TITLE
json x y swapped

### ISSUE THREAD

**Issue by funjoker** [kingdomcomemap/kingdomcomemap.github.io#9]
The json file stores the coords swapped

**Comment by rogerhnn** [comment]
Yeah, it's swapped when it saves, but it loads correctly when you upload it again, I will swap the x and y text soon.
